extension/googleclientauthextension/metadata.yaml

```diff
@@ -9,4 +9,3 @@ status:
 
 tests:
   skip_lifecycle: true # in development
```

Comment:
Should we remove the in development comment?

Reply:
yup done